### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please check for cards component naming if any changes please suggest.

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+.card-list-item .card-content-wrapper {
+    flex-direction: row;
+    flex-grow: 1;
+    gap: 3rem;
+}
+
+.card-list-item .card-content {
+    flex-grow: 1;
+}
+.card-list-item .card-content .card-title {
```
Comment (itsmebipul00) [inline components/card/card.css:124]: you could use a single selector or a new one

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    flex-grow: 1;
+    justify-content: space-between;
+    flex-shrink: 0;
+}
+
+/* card list item */
+
+.card-list-item {
+    flex-direction: row;
+    min-width: 100%;
```
Comment (iRohitGaur) [inline components/card/card.css:110]: any reason for min-width here?

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+.card-title {
+    font-size: 1.8rem;
+}
+
+.card-description {
+    display: -webkit-box;
+    font-size: 1.3rem;
+    color: var(--DARK-GREY);
+    overflow: hidden;
+    /* text-overflow: ellipsis; */
```
Comment (iRohitGaur) [inline components/card/card.css]: remove commented code

File: components/card/card.css (hunk 1 above)
Comment (bhushannaikaroskar) [reply components/card/card.css:124]: Actually its to override the previous one. Let me explain. The basic card-content class is for vertical card. This one is of Card list which is specifically horizontal in shape. So to specify the flex-grow for list cards I have to use multiple selectors

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes 👍🏻

File: components/card/card.css (hunk 2 above)
Comment (bhushannaikaroskar) [reply components/card/card.css:110]: yup actually flex is making it shrink to smaller space I want to make it take the whole available space so used min-width here.

File: components/card/card.css (hunk 3 above)
Comment (bhushannaikaroskar) [reply components/card/card.css]: ok sure

File: components/card/card.css (hunk 2 above)
Comment (iRohitGaur) [reply components/card/card.css:110]: in that case you can use flex-shrink: 0;

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+    flex-shrink: 0;
+}
+
+/* card list item */
+
+.card-list-item {
+    flex-direction: row;
+    min-width: 100%;
+    min-height: 20rem;
+    max-height: 20rem;
```
Comment (iRohitGaur) [inline components/card/card.css]: min-height, max height is not needed if they are the same values. maybe use height directly?

Comment (bhushannaikaroskar) [reply components/card/card.css]: yup sure

File: components/card/card.css (hunk 1 above)
Comment (itsmebipul00) [reply components/card/card.css:124]: Ohh okay, you could have created a new class and used that

**Review by itsmebipul00 — APPROVED** [review]

**Review by Omkar-Ghate — APPROVED** [review]
Looks good for me. Ready to merge

**Pull request merged**